src/graph/test/GoTest.cpp

```diff
@@ -690,5 +691,175 @@ TEST_F(GoTest, is_inCall) {
         ASSERT_TRUE(verifyResult(resp, expected));
     }
 }
+
+
+TEST_F(GoTest, ReverselyOneStep) {
+    {
+        cpp2::ExecutionResponse resp;
+        auto query = "GO FROM hash('Tim Duncan') OVER like REVERSELY "
+                     "YIELD like._src";
+        client_->execute(query, resp);
+        std::vector<std::tuple<int64_t>> expected = {
+            { players_["Tony Parker"].vid() },
+            { players_["Manu Ginobili"].vid() },
+            { players_["LaMarcus Aldridge"].vid() },
+            { players_["Marco Belinelli"].vid() },
+            { players_["Danny Green"].vid() },
+            { players_["Aron Baynes"].vid() },
+            { players_["Boris Diaw"].vid() },
+            { players_["Tiago Splitter"].vid() },
+            { players_["Dejounte Murray"].vid() },
+            { players_["Shaquile O'Neal"].vid() },
+        };
+        ASSERT_TRUE(verifyResult(resp, expected));
+    }
+    {
+        cpp2::ExecutionResponse resp;
+        auto query = "GO FROM hash('Tim Duncan') OVER like REVERSELY "
+                      "YIELD $$.player.name";
```

Comment:
We reference _**$$**_ to indicate the vertex which can go to 'Tim Duncan' here, while we indicate the same thing by referencing _**_src**_ in last test case. Does this logic make sense to users?